Contest_API.md

```diff
@@ -1879,9 +1879,9 @@ Request:
 Returned data:
 
 ```json
-[{"id":"1","from_team_id":"34","to_team_id":null,"reply_to_id":null,"problem_id":null,
+[{"id":"1","from_team_id":"34","to_team_ids":null,"to_group_ids":null,"reply_to_id":null,"problem_id":null,
   "text":"May I ask a question?","time":"2017-06-25T11:59:27.543+01","contest_time":"1:59:27.543"},
- {"id":"2","from_team_id":null,"to_team_id":"34","reply_to_id":"1","problem_id":null,
+ {"id":"2","from_team_id":null,"to_team_ids":["34"],"reply_to_id":"1","problem_id":null,
```

Comment:
Would be good to have an example where there are multiple team IDs and/or a group ID.

Reply:
Agree. Will update.

Reply:
Updated an example to include 3 team ids and a group id in a response.